config/locales/es.yml

```diff
@@ -34,30 +34,30 @@ es:
       authentication:
         disabled: Deshabilitada
         enabled: Habilitada
-        info: Permitir solo a usuarios autenticados ingresar a una sala
+        info: Permitir solo a usuarios autenticados acceder a una sala
         title: Requerir autenticación para hacer uso de salas
-        user-info: Debes iniciar sesión para ingresar a esta sala
+        user-info: Debes iniciar sesión para acceder a esta sala
       branding:
         change: Cambiar imagen
         info: Cambiar la imagen personalizada que aparece en la esquina superior izquierda
         placeholder: Enlace de la imagen...
         title: Imagen personalizada
       cache:
-        info: Limpia el proveedor almacenado en cache lo cual forza todas la actualización de las peticiones nuevas
+        info: Limpia el proveedor almacenado en cache lo cual fuerza todas la actualizaciones de las peticiones nuevas
         title: Limpiar cache de proveedor
         button: Limpiar cache
       color:
-        info: Cambiar el color regular cambiará tanto el color claro como obscuro. Los valores de Aclarar y Oscurecer se pueden cambiar individualmente
+        info: Cambiar el color regular cambiará tanto el color claro como oscuro. Los valores de Aclarar y Oscurecer se pueden cambiar individualmente
         title: Color principal
         regular: Regular
         lighten: Aclarar
         darken: Oscurecer
       recording_visibility:
-        info: Establecer la visibilidad por default en las grabaciones nuevas
-        title: Visibilidad de grabación por default
-        warning: Esta configuración solo será aplicada a las nuevas sesiones
+        info: Establecer la visibilidad por defecto en las nuevas grabaciones
```

Comment:
If we are going to the purest of the language, it should be grabaciones nuevas, as the adjective in Spanish goes after the noun.